components/PDFViewer.jsx

```diff
@@ -29,20 +30,19 @@ const PDFViewer = ({ pdf }) => {
   }, [pdf]);
 
   return (
-    <div className="h-full w-full">
+    <>
       {newPDFLink ? (
-        <iframe
-          className="h-full"
-          src={newPDFLink}
-          height="500vh"
-          width="100%"
-        />
-      ) : (
-        <div className="flex h-full justify-center outline outline-2">
-          <p className="font-lexend text-3xl font-semibold">
-            no resume uploaded
-          </p>
+        <div className="h-full w-full">
+          <iframe
+            className="h-full"
+            src={newPDFLink}
+            height="500vh"
```

Comment:
bro what

Reply:
Is this because of the 500vh?  It was already in the original code and I moved things around but I changed it to 100vh.  Changing the iframe height of a pdf doesn't seem to change the size of it.  The <div> around the object is flex-growing the pdf frame to fill the div. 

Without flex-grow, the iframe is stuck at 1/7ish the screensize

![image](https://github.com/acm-ucr/membership-portal/assets/53993828/1ff12352-072a-4996-a475-484b12431ee5)